test/basic_unified_tests.jl

```diff
@@ -540,7 +549,7 @@ function run_list_test_cases(config::AbstractConfig)
 
         entries = list_objects("list/", config)
         @test length(entries) == 5
-        @test map(x -> x.size, entries) == range(10; step=10, length=5)
+        @test within_margin(map(x -> x.size, entries), range(10; step=10, length=5), 32)
```

Comment:
TODO for future:
when encryption is disabled, test for margin = 0

Reply:
Done